src/co_emcy.c

```diff
@@ -257,6 +257,10 @@ int co_emcy_tx (co_net_t * net, uint16_t code, uint16_t info, uint8_t msef[5])
       net->cb_emcy (net->cb_arg, net->node, code, reg, msef);
```

Comment:
Perhaps the EMCY callback can return a value to tell the stack whether to perform the Error behaviour or not?

Reply:
That seems like a good idea. Do you want to add it to this PR or I can add it in a later commit?

Reply:
Feel free to add to this PR, otherwise I'll do it next time I pick up this work.